tests/console/ncurses.pm

```diff
@@ -24,6 +24,7 @@ use utils qw(clear_console zypper_call);
 sub run {
     select_console 'root-console';
     zypper_call 'in dialog';
+    clear_console;
```

Comment:
It is better to add comments here include the poo id.

Reply:
done. please check it again. thanks